### ISSUE TITLE
Simulation tree view - reactions have an icon for "transporter"

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/MoBi#1149]
![image](https://github.com/Open-Systems-Pharmacology/MoBi/assets/25903040/5ffd8f2f-af7a-4775-a5e1-4586a36b08a7)

"Fluvoxamine-CYP1A2-fit" is actually a reaction.

**Comment by PavelBal** [comment]
[O.1. pop - Omeprazole po 20 mg - DDI without moclobemide - (Cho2002).pkml.zip](https://github.com/Open-Systems-Pharmacology/MoBi/files/12800823/O.1.pop.-.Omeprazole.po.20.mg.-.DDI.without.moclobemide.-.Cho2002.pkml.zip)

**Comment by rwmcintosh** [comment]
This appears to be not new for v12.
I did a similar test in V11 and for Reactions exported from PK-Sim, you get ```icon="Metabolism"```

The icon for metabolism is 
![image](https://github.com/Open-Systems-Pharmacology/MoBi/assets/261477/67ff3b3f-f03b-4b2c-a8d6-ab0efb7447af)

**Comment by msevestre** [comment]
That's correct. This is a metabolism process. Why do you want to change this?

**Comment by msevestre** [comment]
Maybe there is no bug here.

**Comment by rwmcintosh** [comment]
It's not the same project, but does this make more sense? To my knowledge there's no distinct icon for metabolism. This is a duplicate of reaction icon.

![image](https://github.com/Open-Systems-Pharmacology/MoBi/assets/261477/8aaa1158-02ca-4240-ac4d-a40026f98468)

**Comment by PavelBal** [comment]
The icon looks like "transport" for me, clearly misleading.

**Comment by msevestre** [comment]
oh that's not the icon for metabolism lol. So confused here.

**Comment by msevestre** [comment]
> To my knowledge there's no distinct icon for metabolism. This is a duplicate of reaction icon.

but

![image](https://github.com/Open-Systems-Pharmacology/MoBi/assets/1041237/9f7e643c-3fee-4644-9891-4237c3669c19)

so why is the transport icon showing up when metabolism is used?

**Comment by rwmcintosh** [comment]
When I open the metabolism.svg file, that's what I see.
